packages/calypso-css-custom-properties/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Packages shouldn't have locks. The lock is handled by the root.

Reply:
I added it since `npm ci` (when run from this directory) required it. If we don't intend to run it there (in the near future), I guess we can remove it.

Reply:
yeah, you shouldn't be running `npm i` or `npm ci` from inside packages/*. Just the root for now.

Reply:
Removed per 2b03412c82.